vita3k/gui/src/firmware_install_dialog.cpp

```diff
@@ -54,9 +67,22 @@ void draw_firmware_install_dialog(GuiState &gui, HostState &host) {
 
     ImGui::OpenPopup("Firmware Installation");
     if (ImGui::BeginPopupModal("Firmware Installation", nullptr, ImGuiWindowFlags_NoResize)) {
-        ImGui::TextColored(GUI_COLOR_TEXT, "Successfully Installed Firmware");
+        ImGui::TextColored(GUI_COLOR_TEXT, "Firmware successfully installed.");
+        if (!fw_version.empty())
+            ImGui::TextColored(GUI_COLOR_TEXT, "Firmware version: %s", fw_version.c_str());
+        ImGui::Spacing();
+        ImGui::Separator();
+        ImGui::Spacing();
+        ImGui::Checkbox("Delete the firmware installation file?", &delete_fw_file);
+        ImGui::Spacing();
         ImGui::SetCursorPosX(ImGui::GetWindowWidth() / 2 - 30);
         if (ImGui::Button("OK", BUTTON_SIZE)) {
+            if (delete_fw_file) {
+                fs::remove(static_cast<const fs::path>(fw_path));
```

Comment:
```suggestion
                fs::remove(fs::path(fw_path));
```
This is a bit shorter and the cast is probably calling the constructor anyway.